motr/io.c

```diff
@@ -700,6 +700,9 @@ int m0_obj_op(struct m0_obj       *obj,
 		  opcode == M0_OC_FREE ? "free" : "");
 	M0_PRE(obj != NULL);
 	M0_PRE(op != NULL);
+	M0_PRE(ergo(opcode != M0_OC_FREE, attr != NULL));
+	M0_ASSERT(ergo(opcode != M0_OC_FREE, attr->ov_vec.v_count[attr->ov_vec.v_nr - 1]));
+	M0_ASSERT(ergo(opcode != M0_OC_FREE, attr->ov_vec.v_count[0]));
```

Comment:
We should still support attr NULL as some application may not be interested in providing checksum

Reply:
We cannot assume in motr code that attr is always non-NULL and should avoid that assumption and change code accordingly.